ngs_sequence_quality_control/README.md

```diff
@@ -51,77 +50,546 @@ Check conda with the following command:
+
 Run the following commands:
 
     conda config --add channels defaults
     conda config --add channels bioconda
     conda config --add channels conda-forge
 
-*You could also add desired channels by manually editing  `.condarc` file, located in your home folder `~/.condarc`.*
+<details>
+    <summary>shell prompt</summary>
+
+```console
+(base) [lgueguen@n221 demo]$ conda config --add channels defaults
+Warning: 'defaults' already in 'channels' list, moving to the top
+(base) [lgueguen@n221 demo]$ conda config --add channels bioconda
+(base) [lgueguen@n221 demo]$ conda config --add channels conda-forge
+(base) [lgueguen@n221 demo]$ conda info
+
+     active environment : base
+    active env location : /home/fr2424/sib/lgueguen/miniconda3
+            shell level : 1
+       user config file : /home/fr2424/sib/lgueguen/.condarc
+ populated config files : /home/fr2424/sib/lgueguen/.condarc
+          conda version : 4.8.3
+    conda-build version : not installed
+         python version : 3.7.7.final.0
+       virtual packages : __glibc=2.12
+       base environment : /home/fr2424/sib/lgueguen/miniconda3  (writable)
+           channel URLs : https://conda.anaconda.org/conda-forge/linux-64
+                          https://conda.anaconda.org/conda-forge/noarch
+                          https://conda.anaconda.org/bioconda/linux-64
+                          https://conda.anaconda.org/bioconda/noarch
+                          https://repo.anaconda.com/pkgs/main/linux-64
+                          https://repo.anaconda.com/pkgs/main/noarch
+                          https://repo.anaconda.com/pkgs/r/linux-64
+                          https://repo.anaconda.com/pkgs/r/noarch
+          package cache : /home/fr2424/sib/lgueguen/miniconda3/pkgs
+                          /home/fr2424/sib/lgueguen/.conda/pkgs
+       envs directories : /home/fr2424/sib/lgueguen/miniconda3/envs
+                          /home/fr2424/sib/lgueguen/.conda/envs
+               platform : linux-64
+             user-agent : conda/4.8.3 requests/2.24.0 CPython/3.7.7 Linux/2.6.32-431.17.1.el6.x86_64 centos/6.5 glibc/2.12
+                UID:GID : 2369:1000
+             netrc file : None
+           offline mode : False
+```    
+</details>
+
+The `conda config` command updates the `.condarc` file, located in your home folder `~/.condarc`.
+
+    cat ~/.condarc
+
+<details>
+    <summary>shell prompt</summary>
+
+```console
+(base) [lgueguen@n221 demo]$ cat ~/.condarc
+channels:
+  - conda-forge
+  - bioconda
+  - defaults
+```    
+</details>
+
+*You can also add desired channels by manually editing the `.condarc` file.*
 
 ### Search for tool packages
 
-To perform the NGS sequence quality control, you need two tools: FastQC and Cutadapt. Check if Conda packages are available for them, and which version.
+To perform the NGS sequence quality control, you need two tools: FastQC (0.11.9) and Cutadapt (2.10). These are not available in your terminal:
```

Comment:
not or now?

Reply:
not. The conda environments are not installed yet and the tools are note available. The same command (`which) is run before and after the creation of the conda env, to see the difference.

Reply:
true! thanks :)